engines/bladerunner/font.cpp

```diff
@@ -192,6 +233,18 @@ void Font::drawCharacter(const uint8 character, Graphics::Surface &surface, int
 
 	int endY = height + y - 1;
 	int currentY = y;
+
+	// FIXME/TODO
```

Comment:
Not really needed. But it's a safety check; it won't really affect the external (custom) font, and it will prevent crashes with the internal font (mostly when debugging stuff with that font I guess).